pages/en/snapps/how-to-write-a-snapp.mdx

```diff
@@ -257,7 +266,7 @@ class HelloWorld extends SmartContract {
 The `increment()` method in our example is async because it calls `this.x.get()`.
 This is how you can fetch the current state of the snapp account from the chain.
 
-The method then logs the current state, and sets the new state to `x + 1` using `this.x.set()`.
+The method then and sets the new state to `x + 1` using `this.x.set()`.
```

Comment:
@mitschabaude Typo here `The method then and sets`

Reply:
fixed, thanks!